apps/discord-bot/src/commands/minecraft/available.command.ts

```diff
@@ -0,0 +1,80 @@
+import { MojangPlayerArgument } from '#arguments';
+import { MojangApiService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+
+@Command({ description: 'commands.available', args: [new MojangPlayerArgument()] })
+export class AvailableCommand {
```

Comment:
Can you add the information about the name's length on every embed. 

![Screenshot_20220612-080908_Discord.png](https://user-images.githubusercontent.com/42344274/173232430-a167b41b-2e0e-4ca9-aeba-9fe71a2deff3.png)